recipes/isce2/build.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+mkdir $SRC_DIR/{build,install,config,bin}
+
+##Setup a link to cython here
+ln -s $PREFIX/bin/cython $SRC_DIR/bin/cython3
+
+#Setting up some environment variables
+export PATH=$PATH:$PREFIX/bin:$SRC_DIR/bin
```

Comment:
`${PREFIX}/bin` should already be in the `PATH`:
```suggestion
export PATH="${PATH}:${SRC_DIR}/bin"
```